src/styles/main.scss

```diff
@@ -0,0 +1,38 @@
+@import '../styles/components/card';
+@import '../styles/components/stars';
+@import './variables';
+
+$main-color: #00acdc;
+$text-secondary-color: #616070;
+$text-accent-color: #060b35;
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $main-color;
+}
+
+main {
```

Comment:
add such specific styles using class selector instead of tag selector

Also, add max-width to pass all tests

Reply:
i add all fixes like u wrote, but when i start tests, 

I already have 20 errors, not 2 like last time


maybe I'm doing something wrong?
https://prnt.sc/sRrFvRhClNlM